torchvision/transforms/functional.py

```diff
@@ -446,22 +480,29 @@ def _get_perspective_coeffs(startpoints, endpoints):
     return res.squeeze_(1).tolist()
 
 
-def perspective(img, startpoints, endpoints, interpolation=Image.BICUBIC):
+def perspective(img, startpoints, endpoints, interpolation=Image.BICUBIC, fill=None):
     """Perform perspective transform of the given PIL Image.
 
     Args:
         img (PIL Image): Image to be transformed.
         startpoints: List containing [top-left, top-right, bottom-right, bottom-left] of the orignal image
         endpoints: List containing [top-left, top-right, bottom-right, bottom-left] of the transformed image
         interpolation: Default- Image.BICUBIC
+        fill (n-tuple or int or float or None): Pixel fill value for area outside the rotated
```

Comment:
Since `None` simply triggers the default value, IMO we don't need to list it here.

Reply:
I agree, I'll yank it before the next PR.